tasks/main.yml

```diff
@@ -82,11 +82,13 @@
     - https://raw.githubusercontent.com/thoth-station/result-api/master/openshift/buildConfig-template.yaml
     - https://raw.githubusercontent.com/thoth-station/solver/master/openshift/buildConfig-template.yaml
     - https://raw.githubusercontent.com/thoth-station/user-api/master/openshift/buildConfig-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/workload-operator/master/openshift/imageStream-template.yaml
 
 - name: create Thoth Core ImageStream templates
   command: oc apply --namespace "{{ namespace  }}" --filename "{{ item }}"
   with_items:
     - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/imageStream-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/amun-api/master/openshift/imagestream-template.yaml
```

Comment:
ImageStream?

Reply:
The imagestream-template is already merged in amun-api, may be next time we can change the name.